src/components/Popover/util.ts

```diff
@@ -1,13 +1,19 @@
+import { Placement } from '@popperjs/core';
+import { Alignment, Position } from '.';
+
 /**
  * https://popper.js.org/popper-documentation.html#Popper.placements
  *
  * @param {String} position prop value
  * @param {String} alignment prop value
  * @returns {String} valid `placement` value for PopperJS `Popper` component
  */
-export const getPopperPlacement = (position, alignment) => {
+export const getPopperPlacement = (
+  position: Position,
+  alignment: Alignment
+): Placement => {
   // Popperjs does not have a "center" placement variation. It centers by default.
   // Our component explicitly accepts a `center` value for alignment.
   const variation = !alignment || alignment === 'center' ? '' : `-${alignment}`;
-  return `${position}${variation}`;
+  return `${position}${variation}` as Placement;
```

Comment:
[should] I'm generally not a fan of using `as`. Is there another way we can approach this where you don't need to cast this type?

Reply:
I couldn't find one, but let me try something else that I just thought of.

Reply:
ended up removing this file altogether.
- removed `alignment` and `position` props, in exchange we're using `placement` since we're just passing it directly to `Popper`

Reply:
This is a breaking change though, no? We're changing the public api.

Reply:
that's true, you know what, let me do some more research to see what's the best approach that doesn't involve changing the public API, so we're less prone to errors.

Reply:
I think the only alternative this leaves us is, doing 
``const variation: '' | `-${PopperAlignment}`...`` in `component/Popover/util.ts`, if you're ok with this, then so am i.